sdk/core/System.ClientModel/src/ModelReaderWriter/ModelReaderWriter.cs

```diff
@@ -305,4 +310,16 @@ internal static bool ShouldWriteAsJson<T>(IPersistableModel<T> model, ModelReade
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     private static bool IsJsonFormatRequested<T>(IPersistableModel<T> model, ModelReaderWriterOptions options)
         => options.Format == "J" || (options.Format == "W" && model.GetFormatFromOptions(options) == "J");
+
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    private static ModelReaderWriterOptions GetOptions(ModelReaderWriterOptions? options)
+    {
+        if (options is null)
+            return ModelReaderWriterOptions.Json;
```

Comment:
this is safe because the default won't have proxies, so the `ProxiedModel` property never comes into play, right?

Reply:
Correct, we just use the static reference here for default which has no proxies.